oauth2client/contrib/django_util/signals.py

```diff
@@ -20,7 +20,7 @@
 
 import django.dispatch
 
-"""Signal that fires when  OAuth2 Flow has completed.
+""" Signal that fires when  OAuth2 Flow has completed.
```

Comment:
What's the significance of this change? If there's any change that should be made to this line, shouldn't it be to change the double-space into a single-space?